### ISSUE TITLE
Grav 1.1.0 rc1 admin left sidebar enlarging again after each page save

### ISSUE THREAD

**Issue by danielpetre** [getgrav/grav#890]
Hi, so if i minimize the left sidebar admin menu and i edit a page and save.. the left menu is enlarging itself again.

**Comment by rhukster** [comment]
Yes it's currently only a page-level thing.  There is no memory for it. I'll leave a note for @w00fz as this will require some JS magic to keep track of sidebar status.
